src/ered_cluster.erl

```diff
@@ -491,31 +473,28 @@ handle_info({slot_info, Version, Response, FromAddr}, State) ->
                                        clients = Clients,
                                        slot_map_version = Version + 1,
                                        slot_map = NewMap,
-                                       convergence_check = nok,
+                                       slot_map_from = FromAddr,
+                                       convergence_check = not_started,
                                        masters = MasterNodes,
                                        closing = NewClosing},
                     {noreply, update_cluster_state(State2)}
-            end
+            end;
+        Unexpected ->
+            report_unexpected_slots_response(Unexpected, FromAddr, State#st.info_pid),
+            {noreply, State#st{slot_map_from = none}}
```

Comment:
Do we need to reset `slot_map_from` now? Shouldn't it just be set when the slot_map is set?